src/app/app-routing.module.ts

```diff
@@ -13,6 +14,7 @@ const routes: Routes = [
   {path: '' , component: IndexComponent},
   {path:'itinerarios/:id', component: ItinerarylistComponent,pathMatch: 'full'},
   {path:'mis_itinerarios/:id', component: MyItinirariesComponent,pathMatch: 'full'},
+  {path:'itinerario/:id', component: ItineraryViewContoller,pathMatch: 'full'},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
  {path:"itinerario/:id", component: ItineraryViewContoller,pathMatch: 'full'},
```